VideoWeb/VideoWeb.Common/SignalR/AdUserProfileService.cs

```diff
@@ -1,30 +1,41 @@
 using System.Threading.Tasks;
 using UserApi.Client;
 using UserApi.Contract.Responses;
+using VideoApi.Client;
+using VideoApi.Contract.Consts;
+using VideoApi.Contract.Enums;
 
 namespace VideoWeb.Common.SignalR
 {
     public interface IUserProfileService
     {
-        Task<string> GetObfuscatedUsernameAsync(string username);
+        Task<string> GetObfuscatedUsernameAsync(string participantUserName);
         Task<UserProfile> GetUserAsync(string username);
+
     }
 
     public class AdUserProfileService : IUserProfileService
     {
         private readonly IUserApiClient _userApiClient;
+        private readonly IVideoApiClient _videoApiClient;
 
-        public AdUserProfileService(IUserApiClient userApiClient)
+        public AdUserProfileService(IUserApiClient userApiClient, IVideoApiClient videoApiClient)
         {
             _userApiClient = userApiClient;
+            _videoApiClient = videoApiClient;
         }
 
-        public async Task<string> GetObfuscatedUsernameAsync(string username)
+        public async Task<string> GetObfuscatedUsernameAsync(string participantUserName)
         {
             try
             {
-                var profile = await _userApiClient.GetUserByAdUserNameAsync(username);
-                var userName = profile.FirstName + " " + profile.LastName;
+                var userName = participantUserName;
+                if (!participantUserName.EndsWith(QuickLinkParticipantConst.Domain))
+                {
+                    var profile = await _userApiClient.GetUserByAdUserNameAsync(participantUserName);
+                    userName = profile.FirstName + " " + profile.LastName;
```

Comment:
I know this was already implemented like this but can you try using string interpolation here instead: 
```suggestion
                    userName = $"{profile.FirstName} ${profile.LastName}";
```

Make sure you write the test first (or ensure there are tests available) to ensure we are not changing the behavior.